cmd/config/get.go

```diff
@@ -0,0 +1,57 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+	"reflect"
+
+	"github.com/spf13/cobra"
+)
+
+func NewGetCommand(ctx *ConfigCommandContext) *cobra.Command {
+	getCommand := &cobra.Command{
+		Use:   "get",
+		Short: "Get configuration value",
+		Long:  `Get value for specific key from datree config.yaml file`,
+		Run: func(cmd *cobra.Command, args []string) {
+			messages := ctx.Messager.LoadVersionMessages(ctx.CliVersion)
+
+			err := get(ctx, args[0])
+			if err != nil {
+				fmt.Printf("Failed getting %s . Error: %s", args[0], err)
+			}
+
+			msg, ok := <-messages
+			if ok {
+				ctx.Printer.PrintMessage(msg.MessageText+"\n", msg.MessageColor)
+			}
+		},
+		Args: func(cmd *cobra.Command, args []string) error {
+			if len(args) != 1 {
+				return errors.New("requires exactly 1 arguments")
+			}
+
+			validKeys := make(map[string]bool)
+			validKeys["token"] = true
+
+			if val, ok := validKeys[args[0]]; !ok || !val {
```

Comment:
I think since this validation also appears on the `set` command, can we outsource this validation as a function and use it in both commands?

@dimabru, your thoughts?

Reply:
I agree. Adding 2 `validKeys` objects that we need to maintain is bad practice. I think its best to keep the `validKeys` logic in one place to avoid potential bugs should we later decide to add/remove keys in config.yaml.
@vbh what do you think?